pkg/endpointmanager/conntrack.go

```diff
@@ -71,72 +71,66 @@ func runGC(e *endpoint.Endpoint, isIPv6 bool, filter *ctmap.GCFilter) {
 	}
 }
 
-func createGCFilter(ipv6, initialScan bool, restoredEndpoints []*endpoint.Endpoint) *ctmap.GCFilter {
+// scrubAllNonRestoredIPs runs only when the cilium-agent starts, this function
+// scrub all IPs from the conntrack table which do not belong to IPs of any
+// endpoint that has been restored. No new endpoints can appear yet so we can
+// assume that any other entry not belonging to a restored endpoint has become
+// stale.
+func scrubAllNonRestoredIPs(ipv6 bool, restoredEndpoints []*endpoint.Endpoint) {
```

Comment:
This description is wrong. The function is called on every gc run and is responsible to do GC on per endpoint CT tables. The condition is: `len(eps) > 0 || initialScan`